packages/shared/src/viewer/helpers/state.ts

```diff
@@ -73,6 +86,10 @@ export type SerializedViewerState = {
     }
     explodeFactor: number
     selection: Nullable<number[]>
+    measurements: Nullable<{
```

Comment:
`enable` doesn't need to be nullable, it can be false by default. Otherwise you're introducing extra complexity - not only do you have to think about what happens when `enable` is true or false, but also what happens if its `null`

IMO `enable` should be a bool, and the options can be nullable.

Reply:
Also in the prop name `enableMeasurements`, the `Measurements` suffix is redundant, it's already clear that its about measurements from the parent property name